packages/kafka/package.json

```diff
@@ -44,14 +44,14 @@
         "prepublishOnly": "npm run lint && npm run build"
     },
     "dependencies": {
```

Comment:
let's also add engine entry, matching the platformatic kafka one, to clearly communicate requirements, and also probably mention it in the readme